docs/module_docs/featdet_resolver_local_minimum/local-minimum-resolver.md

```diff
@@ -1,71 +1,82 @@
-parameters with a noisy EIC. Most of the time, a noisy EIC can be found by sorting the feature table
-by decreasing area.
+During the EICs building, overalpping and partially co-eluting peaks are retained as single features in the faeture list (see, for example, [ADAP chromatogram builder](../featdet_adap_chromatogram_builder/adap-chromatogram-builder.md)). The _local minimum resolver_ module aims at splitting such "shoulder" LC peaks into individual features (_i.e._ [chromatographic deconvolution](../../terminology/general-terminology.md#chromatographic-deconvolution)) based on local minima. In fact, a local minimum in the EIC trace might correspond to the valley between two adjacent, partially-resolved peaks.
 
-![](chromatogramresolving.png)
+The algorithm examines all the data points in the EIC trace starting from the earliest RT. A scan window is set (see _Minimum search range RT/Mobility_ parameter) and centred around the examined data point. A data point is considered a local minimum if it is the lowest intense point within the scan window. Moreover, it has to be X times lower than the highest point in the window (see _Min ratio of peak top/edge_ parameter). When a local minumum is found, a set of user-defined intensity and peak duration requirements is checked. If they are fulfilled, the original overlapping peaks are splitted into new, distinct features.
 
-### Parameter settings
+With the implementation of IM support in MZmine 3, this module was expanded and can now be applied to both the RT and IM dimension (see [Resolving the ion mobility dimension](#resolving-the-ion-mobility-dimension)).
 
-##### MS/MS scan paring
+:octicons-light-bulb-16: **Tip**. The LMR is particularly suitable for LC-MS data with little 
+noise and nice peak shapes.
 
-Selecting this parameter will pair DDA MS/MS spectra to the resolved features. This is optional at
-this stage, because it will be executed again during resolving in the mobility dimension.
+---
+## **Parameters settings**
+:material-menu-open: Feature detection → Chromatogram resolving → Local minimum resolver
 
-##### Dimension
+![](local-minimum-resolver.png)
 
-The dimension to be resolved can be selected here. Select _Retention time_.
 
-##### Chromatographic threshold
+#### **Suffix**
+String added as suffix when creating the corresponding feature list.
+
+#### **Original feature list**
+Keep or remove the processed feature list(s).
+
+#### **MS/MS scan pairing**
+Pair MS/MS fragmentation spectra collected in [DDA](../../terminology/general-terminology.md#data-dependent-acqusition-mode) mode to the resolved features. This is optional at this stage as the same can be done with a separate [module](../featdet_ms2_scan_pairing/ms2_scan_pairing.md). See [MS2 scan pairing](../featdet_ms2_scan_pairing/ms2_scan_pairing.md) documentation for more details.
+
+
+#### **Dimension**
+The dimension to be resolved can be selected here. Select _Retention time_. ... 
```

Comment:
what are the ... for?

Reply:
Fixed. Still need to finalize the doc of this module though. Will do in the next days. If you want I can do it before merging. So we'll have Mass detection, ADAP and Local Minimum Resolver modules ready.